core/src/components/router/utils/dom.ts

```diff
@@ -3,6 +3,12 @@ import { componentOnReady } from '../../../utils/helpers';
 
 import { ROUTER_INTENT_NONE } from './constants';
 
+// Activates the passed route chain.
```

Comment:
These comment blocks will need to be converted to multi-line: `/**`. For whatever reason, VSCode doesn't detect these code blocks for the tooltip hover interaction otherwise. 

This applies to most root-level comments. Local variables that aren't exported don't _have to_ have multi-line, but it is a project convention we should try to follow.

Reply:
I completely overlook hiver interaction, sorry.

One other benefit of JSDoc is that it makes it easier to document parameters and return values. I'll keep improving the doc blocks in the follow up PRs.

(I have pushed an updated version)